source/_patterns/04-components/captioned-image/captioned-image.twig

```diff
@@ -3,7 +3,7 @@
   modifier_classes ? modifier_classes,
 ]|join(' ')|trim %}
 
-<figure class="{{ classes }}">
+<figure class="{{ classes }}"{% if caption %} role="group"{% endif %}>
```

Comment:
Why is `role="group"` added here?

Reply:
I added role=group because that's in the original stable theme's file (filter-caption.html.twig)